core/nandtool/nandtool.c

```diff
@@ -128,7 +140,7 @@ static int nandtool_flash(const char *path, unsigned int start, int raw)
 }
 
 
-static int nandtool_erase(unsigned int start, unsigned int size)
+static int nandtool_erase(unsigned int start, unsigned int size, int write_cleanmarkers)
```

Comment:
Nitpick/coding style - use camelCase
```suggestion
static int nandtool_erase(unsigned int start, unsigned int size, int writeCleanMarkers)
```

Reply:
leaving as-is because of current coding style of the code